tests/Makefile.am

```diff
@@ -200,6 +200,8 @@ check_PROGRAMS = $(PURE_EXECUTABLES) \
 	ioctl_kd-success-s1024-Xabbrev \
 	ioctl_kd-success-s1024-Xraw \
 	ioctl_kd-success-s1024-Xverbose \
+	ioctl_lirc \
```

Comment:
ioctl_lirc is redundant in this list because it's listed in pure_executables.list;
ioctl_lirc-success has to be kept here, see below.

Reply:
done